pkg/endpoint/endpoint.go

```diff
@@ -453,7 +453,7 @@ func (e *Endpoint) GetModelRLocked() *models.Endpoint {
 // newHealthz returns the endpoint's healthz object.
 //
 // Must be called with e.Mutex locked.
-func (e *Endpoint) getHealthzModel() *models.EndpointHealthz {
+func (e *Endpoint) GetHealthzModel() *models.EndpointHealthz {
```

Comment:
Create a public method that can be called without the e.Mutex locked. You are calling this function without locking the endpoint in daemon/endpoint.go:581

Reply:
You are correct. fixed.